packages/dashboards/addon/components/dashboard-header.hbs

```diff
@@ -0,0 +1,130 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="container-full m-b-4">
+  <div class="row m-t-10">
+    <div class="xs-col-1-1 sm-col-1-2 col-1-5 p-x-0">
```

Comment:
Can this be?
```suggestion
    <div class="col-1-1">
```

Reply:
![Screen Shot 2021-02-25 at 9 46 28 AM](https://user-images.githubusercontent.com/59849444/109169990-5f013080-774e-11eb-90db-b5d7da2a29a0.png)
This happens

Reply:
You can probably just take off the icon. `DenaliBreadcrumb` does not include that: 

https://denali-design.github.io/denali-ember/?path=/story/denalibreadcrumb--playground